plugins/org.jboss.tools.openshift.client/.classpath

```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
```

Comment:
I would highly recommend not to remove this, it wont get regenerated automatically on import (especially the java build path exports)